locale/fr.po

```diff
@@ -45,7 +46,7 @@ msgstr "Erreur inconnue"
 
 #: crashreports/crashreporter/CrashReportApp.cpp
 msgid "Problem Report for Audacity"
-msgstr "Rapport de problème pour Audacity"
+msgstr ""
```

Comment:
I wonder why this got empty :-)

Reply:
Because the guys that receive this email, doen't speak french ;-)

Reply:
Well, we can use Google Translate. 

On top of that - Crash Reporter sends out a binary with a stack trace, no user input is allowed